geo-specific-codelist/parliamentary-geography.cypher

```diff
@@ -0,0 +1,1334 @@
+CREATE CONSTRAINT ON (n:`_code_parliamentary-geography`) ASSERT n.code IS UNIQUE;
+CREATE (node:`_code_list`:`_code_list_parliamentary-geography` { label:"geography", edition:"one-off" });
+MERGE (node:`_code`:`_code_parliamentary-geography` { value:"E14000530" });
+MATCH (parent:`_code_list`:`_code_list_parliamentary-geography`),                (node:`_code`:`_code_parliamentary-geography` { value:"E14000530" })                 MERGE (node)-[:usedBy { label:"Aldershot"}]->(parent);
```

Comment:
Code labels for geography should use the `_code_geography` label instead of a list specific label to avoid duplicates/enable geography journies